java/src/main/java/com/saucelabs/saucebindings/SystemManager.java

```diff
@@ -34,4 +38,20 @@ public static String get(String key) {
             return null;
         }
     }
+
+    public static String getCurrentGitBranch() {
```

Comment:
What's the use case of this?

Reply:
To populate the `branch` value. If a value is not set, screener defaults to `"(default)"` (with the parentheses) which breaks some url renderings. This matches the behavior of screener with CLI, etc, and users can always override this value.